provider/k8s/helpers.go

```diff
@@ -314,3 +315,27 @@ func volumeTo(v string) (string, error) {
 		return "", errors.WithStack(fmt.Errorf("invalid volume %q", v))
 	}
 }
+
+func toCpuCore(millicore int64) float64 {
+	// 1000m (milicores) = 1 core = 1 vCPU = 1 AWS vCPU = 1 GCP Core
+	return float64(millicore) / 1000
+}
+
+func toMemMB(bytes int64) float64 {
+	// 1024*1024 bytes = 1 MiB
+	return float64(bytes) / (1024 * 1024)
+}
+
+func calculatePodCpuAndMem(m *metricsv1beta1.PodMetrics) (cpu float64, mem float64) {
```

Comment:
This function is not meant to be a helper, it's a provider function, perhaps it's better to move it to be a Provider function, something like `p.CalculatePodCpuAndMem`. Since you have the metrics interface in the Provider struct, you won't need to pass it to the function.

Reply:
`m *metricsv1beta1.PodMetrics` this parameter is struct, it contains containers usage metrics. it differs pod to pod, so i have to pass it as parameter. It basically sums the resource usage and return it. That's why i put it in the helpers method.
But it can be made provider function too, but the parameter still will be there.
Should i make it provider function, what's your call?

Reply:
No need, leave as it is 👍